### DISCUSSION TITLE
Can't trigger onBlur in Controller

### DISCUSSION THREAD

**Discussion by WeakCookie** [react-hook-form/react-hook-form#4335] [Q&A] [answered]
Hi! I'm using `Controller` with Select from react-select, every prop is passed from `Controller` to `Select` properly except for the `onBlur` one, it doesn't even trigger the `onBlur`
My code look something like this

```
    <Controller
          as={<Select />}
          onBlur={(e) => console.log(e)} // not thing happen !!!
          name={'name'}
          control={control}
          onChange={([selected]) => {
            return selected
          }}
          options={options}
     />
```

I have also used `render` but it doesn't seem to work :(
Am i missing or doing anything wrong, please let me know. Thanks in advance

**Comment by lop862** [comment]
Density: 280 dp / xhdpi / 1.75x
Exact Density: 277.09 * 274.394 dp
Resolution: 720 * 1480 px
Available Resolution: 720 * 1396 px
Screen Size: 2.6" * 5.4" / 6.0 Inches una aplicacion

**Comment by bluebill1049** [comment, marked as answer]
```
    <Controller
          render={(props) => <Select {...props} />}
          name={'name'}
          control={control}
          options={options}
     />
```

can you try above? and which version are you using? if still not working, create a simple csb, happy to take a look at it.

**Reply by WeakCookie** [reply]
I'm using 6.9.6, and the above doesn't even pass down the options :(, i will update a csb asap

**Reply by bluebill1049** [reply]
```
    <Controller
          render={(props) => <Select {...props} options={options} />}
          name={'name'}
          control={control}
     />
```

my bad... above should fix it

**Reply by WeakCookie** [reply]
https://codesandbox.io/s/react-hook-form-controller-onblur-not-called-forked-z1h16?file=/src/index.js
here is a little sandbox i have created

**Reply by bluebill1049** [reply]
https://codesandbox.io/s/react-hook-form-controller-onblur-not-called-forked-y0d2h?file=/src/index.js

**Reply by WeakCookie** [reply]
oh it worked ! thanks for your response, really appreciate

**Reply by bluebill1049** [reply]
no worries 👍

**Reply by bluebill1049** [reply]
always use `render` 👍 `as` will get deprecated next version
